apps/governance/src/components/telemetry-dialog/telemetry-dialog.tsx

```diff
@@ -0,0 +1,129 @@
+import { create } from 'zustand';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocalStorage } from '@vegaprotocol/react-helpers';
+import { Dialog, Icon, Button } from '@vegaprotocol/ui-toolkit';
+import { Networks, useEnvironment } from '@vegaprotocol/environment';
+
+type TelemetryDialogState = {
+  isOpen: boolean;
+  open: () => void;
+  close: () => void;
+};
+
+const useTelemetryDialogStore = create<TelemetryDialogState>((set) => ({
```

Comment:
chore (non blocking): For typescrip zustand docs recommend `create<T>()((set) => ...)`